docs/EndToEndTests/Settings.md

```diff
@@ -0,0 +1,56 @@
+## Settings
+
+- [Settings](#settings)
+  - [Overview](#overview)
+- [Language](#language)
+  - [Test Objectives](#test-objectives)
+  - [Special Considerations](#special-considerations)
+- [Coordinates](#coordinates)
+  - [Test Objectives](#test-objectives-1)
+  - [Special Considerations](#special-considerations-1)
+- [About Screen](#about-screen)
+  - [Test Objectives](#test-objectives-2)
+  - [Special Considerations](#special-considerations-2)
+
+## Language
+
+- Language button enables user to change the language in the app
+
+### Test Objectives
+
+- Clicking on App Settings shows option to change language
+- Clicking on a particular language changes the language in the app
+- Clicking back to English (or another language) changes it again
+
+### Special Considerations
+
+- A lot of the translations haven't been done yet. Ideally this test will be added to when the translations are done to check the language changes throughout the app. In each script there is a call to check the language, but for testing purposes it will be much more efficient to just check the language changes in one test.
+
+## Coordinates
+
+- Coordinate button enables user to see and change the coordinate formats
+
+### Test Objectives
+
+- Clicking on App Settings shows option to change coordinate system
+- There are three coordinate systems visible
+- Back buttons work to return to settings screen
+
+### Special Considerations
+
+- It is not possible within Maestro to test the functioning of the Radio buttons, although theoretically I should have been able to. Settings is spelled setttings in the screen...
```

Comment:
> Settings is spelled setttings in the screen

this seems like it's worth fixing in this PR, given that we're writing tests that check that.

Reply:
There will be several file changes!

Reply:
ok then a separate dedicated PR is preferable?

Reply:
> There will be several file changes!

generally not worried about this if the nature of the changes are straightforward (i would imagine they are when it comes to a copy fix)